test/specs/platform.basic.tests.js

```diff
@@ -0,0 +1,93 @@
+import BasicChartWebWorker from 'web-worker:../BasicChartWebWorker';
+
+describe('Platform.basic', function() {
+
+	it('should automatically choose the BasicPlatform for offscreen canvas', function() {
+		const chart = acquireChart({type: 'line'}, {useOffscreenCanvas: true});
+
+		expect(chart.platform).toBeInstanceOf(Chart.platforms.BasicPlatform);
+
+		chart.destroy();
+	});
+
+	fit('supports choosing the BasicPlatform in a web worker', function(done) {
+		const canvas = document.createElement('canvas');
+		if (!canvas.transferControlToOffscreen) {
+			pending();
+		}
+		const offscreenCanvas = canvas.transferControlToOffscreen();
+
+		const worker = new BasicChartWebWorker();
+		worker.onmessage = (event) => {
```

Comment:
The `BasicChartWebWorker.js` file defines an `onmessage`. Is this overriding it? Why?